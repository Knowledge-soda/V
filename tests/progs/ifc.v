f(x, y) := +(x, y)
g(x, y) := *(x, y)
print(call(f, 12, 8))
print(ifc(2, f, 12, 8))
print(ifc(0, f, 12, 8))
print(ifec(2, f, g, 12, 8))
print(ifec(0, f, g, 12, 8))
