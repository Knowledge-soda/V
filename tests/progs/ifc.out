20
20
8
20
96
